ui/shared/BugFiler.jsx

```diff
@@ -177,6 +178,10 @@ export class BugFilerClass extends React.Component {
     };
   }
 
+  componentDidMount() {
+    this.checkForSecurityIssue();
```

Comment:
Why is this called when the component mounts?

Reply:
When the bugfiler gets rendered with an initial summary which indicates a security issue, the summary needs to be evaluated for this (comment is always blank on render).